pynumaflow/function/server.py

```diff
@@ -164,7 +166,9 @@ def MapTFn(
             )
         except Exception as err:
             _LOGGER.critical("UDFError, re-raising the error: %r", err, exc_info=True)
-            raise err
```

Comment:
Not sure why simply raise the error won't work in this case?

Reply:
we are trying to generalize this https://github.com/numaproj/numaflow/issues/232#issuecomment-1513286370